### DISCUSSION TITLE
Allow temporarily popping out inspector widgets in a floating window

### DISCUSSION THREAD

**Discussion by mieldepoche** [godotengine/godot-proposals#7245] [Editor]
![image](https://github.com/godotengine/godot-proposals/assets/71937306/1d07ae55-1f54-44b7-a24a-e80b8d6c7190)
My inspector is huge and this curve is still a pain to edit. I wish I could pop that widget out in its own bigger window for editing.
idk if this would work in practice, maybe this could be useful for gradients too?
I have a feeling this can be done with an addon, maybe I'll look into it...

what do you think?

**Comment by aXu-AP** [comment]
Second possibility would be bottom dock, like with animations or tilemaps.
Edit: Text fields (for example in Label) have button for popup window, so I guess this is kind of already implemented. But I agree, curves and gradients would benefit from similiar feature.
